frontend/src/components/admin/AddNewUserPanel.js

```diff
@@ -0,0 +1,204 @@
+import React from 'react';
+import { useState } from 'react'
+import { makeStyles } from '@material-ui/core/styles';
+import InputLabel from '@material-ui/core/InputLabel';
+import FormControl from '@material-ui/core/FormControl';
+import TextField from '@material-ui/core/TextField';
+import Grid from '@material-ui/core/Grid';
+import { Button, Card, CardContent, CardHeader, MenuItem, Select, Typography } from '@material-ui/core';
+
+
+const faculites = [
+    'Wydział Górnictwa i Geoinżynierii',
+    'Wydział Inżynierii Metali i Informatyki Przemysłowej',
+    'Wydział Elektrotechniki, Automatyki, Informatyki i Inżynierii Biomedycznej',
+    'Wydział Informatyki, Elektroniki i Telekomunikacji',
+    'Wydział Inżynierii Mechanicznej i Robotyki',
+    'Wydział Geologii, Geofizyki i Ochrony Środowiska',
+    'Wydział Geodezji Górniczej i Inżynierii Środowiska',
+    'Wydział Inżynierii Materiałowej i Ceramiki',
+    'Wydział Odlewnictwa',
+    'Wydział Metali Nieżelaznych',
+    'Wydział Wiertnictwa, Nafty i Gazu',
+    'Wydział Zarządzania',
+    'Wydział Energetyki i Paliw',
+    'Wydział Fizyki i Informatyki Stosowanej',
+    'Wydział Matematyki Stosowanej',
+    'Wydział Humanistyczny'
+]
+
+const useStyles = makeStyles((theme) => ({
+    margin: {
+        margin: theme.spacing(1),
+    },
+}));
+
+export default function AddNewUserPanel() {
+    const [newUserData, setNewUserData] = useState({
+        'first_name': null,
+        'last_name': null,
+        'faculty': null,
```

Comment:
Email i faculty w bazie danych są zamienione miejscami
`fields = ('first_name', 'last_name', 'email', 'faculty', 'is_staff')`
Nie wiem, czy ma to znaczenie?

Reply:
nie ma :)